jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/coder/DecoderHelper.java

```diff
@@ -0,0 +1,107 @@
+package us.dot.its.jpo.ode.coder;
+
+import java.io.InputStream;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import gov.usdot.asn1.generated.ieee1609dot2.ieee1609dot2.Ieee1609Dot2Data;
+import gov.usdot.cv.security.msg.IEEE1609p2Message;
+import us.dot.its.jpo.ode.model.OdeData;
+import us.dot.its.jpo.ode.model.OdeObject;
+import us.dot.its.jpo.ode.model.SerialId;
+import us.dot.its.jpo.ode.plugin.j2735.J2735Bsm;
+import us.dot.its.jpo.ode.plugin.j2735.J2735MessageFrame;
+import us.dot.its.jpo.ode.plugin.j2735.oss.Oss1609dot2Coder;
+import us.dot.its.jpo.ode.plugin.j2735.oss.OssJ2735Coder;
+import us.dot.its.jpo.ode.security.SecurityManager;
+import us.dot.its.jpo.ode.security.SecurityManager.SecurityManagerException;
+
+public class DecoderHelper {
+   
+   private static final Logger logger = LoggerFactory.getLogger(DecoderHelper.class);
+   
+   private OssJ2735Coder j2735Coder;
+
+   private Oss1609dot2Coder ieee1609dotCoder;
+
+   public DecoderHelper(OssJ2735Coder jCoder, Oss1609dot2Coder ieCoder) {
```

Comment:
Helper classes usually contain static methods only. Why not allow the Helper instantiate the coders as static fields so we won't have to pass them to the Helper object and no need for a constructor.